lib/src/content/components/main_content/wolt_modal_sheet_top_bar_title_flow.dart

```diff
@@ -49,34 +43,30 @@ class _TopBarTitleFlowDelegate extends FlowDelegate {
   final double heroImageHeight;
   final ValueListenable<double> currentScrollPositionListenable;
   final GlobalKey titleKey;
-  final double pageTitlePaddingTop;
   final BuildContext buildContext;
 
   _TopBarTitleFlowDelegate({
     required this.topBarHeight,
     required this.heroImageHeight,
     required this.currentScrollPositionListenable,
     required this.titleKey,
-    required this.pageTitlePaddingTop,
     required this.buildContext,
   }) : super(repaint: currentScrollPositionListenable);
 
   double get currentScrollPosition => currentScrollPositionListenable.value;
 
   @override
   void paintChildren(FlowPaintingContext context) {
-    double pageTitleHeight = titleKey.currentContext!.size!.height;
+    final pageTitleHeight = titleKey.currentContext!.size!.height;
```

Comment:
Preserve type. `final double`

Reply:
Can you clarify this? what is the advantage?

Reply:
done in here: 2959534abffa3d68c6fec75a0c330019920eb9a6

Reply:
I'm gonna do this in separate PRs myself. Add strict rules for better format, types and imports.

When assigning values the types makes sure correct value is assigned. `double` is always`, `int` is always. Unless the value is assigned and never updated you can ommit 

`titleKey.currentContext!.size!.height;` is something that might change so i would mention type.

Reply:
You read more [here ](https://dart.dev/effective-dart/design#do-type-annotate-fields-and-top-level-variables-if-the-type-isnt-obvious)